docs/how-to-guides/admin/deploying-renku.rst

```diff
@@ -59,45 +59,36 @@ This response basically means that:
 Officially supported Kubernetes versions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The following component versions have been tested in production.
+Renku versions ``0.6.x`` and older only work with Helm version 2.
 
-===================   ============   ============
-Renku version         Kubernetes     Helm
-===================   ============   ============
-0.6.x                 1.14, 1.15     2
-0.7.x                 1.16, 1.17     3
-0.8.x                 1.19, 1.20     3
-0.12.x                1.20, 1.21     3
-===================   ============   ============
+Newer versions of Renku only work with Helm version 3 and should be supported
+on the latest version of Kubernetes and 3-4 older minor versions.
 
 Pre-deployment steps
 -----------------------
 
-1. (Optional) Using external GitLab and/or Keycloak
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+1. Setting up Gitlab
+~~~~~~~~~~~~~~~~~~~~
 
 .. toctree::
    :maxdepth: 1
 
    GitLab <configurations/external-gitlab>
-   Keycloak <configurations/external-keycloak>
 
-1. (Optional) Create and configure Renku PVs and PVCs
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+1. (Optional) Using external Keycloak
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-All of Renku information is stored in three volumes needed by Jena, Postgresql
-and GitLab (if not external). You can leave k8s to dynamically provision
-these volumes but we advise to create them beforehand and make regular backups.
-You can use the following manifest files as examples.
+.. toctree::
+   :maxdepth: 1
 
-   - `Renku PV yaml file <https://github.com/SwissDataScienceCenter/renku-admin-docs/blob/master/renku-pv.yaml>`_
-   - `Renku PVC yaml file <https://github.com/SwissDataScienceCenter/renku-admin-docs/blob/master/renku-pvc.yaml>`_
+   Keycloak <configurations/external-keycloak>
 
-.. note::
-   These examples are specific to our use of OpenStack and the details of
-   the storage classes and the volume provider will be different depending on the
-   specifics of your cluster.
+1. (Optional) Create and configure Renku PVs and PVCs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+All of Renku information is stored in two volumes needed by Jena and Postgresql. 
+You can leave k8s to dynamically provision these volumes but we advise to create them 
```

Comment:
```suggestion
You can leave Kubernetes to dynamically provision these volumes but we advise to create them 
```
In the docs I'd rather use full names to avoid confusion from less experienced users/admins.